tests/csm/rest/test_capacity_usage.py

```diff
@@ -1084,3 +1084,423 @@ def test_39923(self):
+
+        total_written = s3_misc.get_total_used(self.akey, self.skey)
+
+        self.log.info("[Start] Start some IOs")
+        obj = f"object{self.s3_user}{time.time_ns()}.txt"
+        self.log.info("Verify Perform %s of %s MB write in the bucket: %s", obj, self.aligned_size,
+                        self.bucket)
+        resp = s3_misc.create_put_objects(
+            obj, self.bucket, self.akey, self.skey, object_size=self.aligned_size)
+        assert resp, "Put object Failed"
+        self.log.info("[End] Start some IOs")
+
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+        time.sleep(self.update_seconds)
+        self.log.info("[Start] Sleep %s", self.update_seconds)
+
+        resp = self.csm_obj.get_degraded_all(self.csm_obj.hlth_master)
+
+        new_write = self.aligned_size * 1024 * 1024
+        total_written += new_write
+
+        result = self.csm_obj.verify_degraded_capacity(resp, healthy=total_written, degraded=0,
+            critical=0, damaged=0, err_margin=self.err_margin, total=total_written)
+        assert result[0], result[1]
+        self.log.info("[END] Setup Method")
+
+    def teardown_method(self):
+        """
+        Teardowm method for deleting s3 account created in setup.
+        """
+        self.log.info("[START] Teardown Method")
+        if self.restore_pod:
+            self.log.info("Failed deployments : %s", self.failed_pod)
+            for deploy_name in self.failed_pod:
+                self.log.info("[Start]  Restore deleted pods : %s", deploy_name)
+                resp = self.csm_obj.master.create_pod_replicas(num_replica=1, deploy=deploy_name)
+                self.log.debug("Response: %s", resp)
+                assert resp[0], f"Failed to restore pod by {self.restore_method} way"
+                self.log.info("Successfully restored pod by %s way", self.restore_method)
+                self.log.info("[End] Restore deleted pods : %s", deploy_name)
+
+        if self.s3_cleanup:
+            self.log.info("Deleting bucket %s & associated objects", self.bucket)
+            resp = s3_misc.delete_all_buckets(self.akey,self.skey)
+            assert resp, "Delete all buckets and object failed."
+            self.log.info("Deleting S3 account %s created in setup", self.s3_user)
+            resp = self.csm_obj.delete_s3_account_user(self.s3_user)
+            assert resp.status_code == HTTPStatus.OK, "Failed to delete S3 user"
+
+        if not self.deploy:
+            bundle_dir = os.path.join(LOG_DIR, "latest", "support_bundle")
+            self.log.info("Support bundle dir : %s", bundle_dir)
+            resp = sb.collect_support_bundle_k8s(local_dir_path=bundle_dir,
+                                                scripts_path=K8S_SCRIPTS_PATH)
+
+        self.deploy = True
+        if self.deploy:
+            self.log.info("Cleanup: Destroying the cluster ")
+            resp = self.deploy_lc_obj.destroy_setup(self.csm_obj.master, self.node_list,
```

Comment:
self.node_list is same as worker_list in test_lib

Reply:
replaced

Reply:
done